vendor/bat-native-ledger/src/bat/ledger/internal/common/time_util.h

```diff
@@ -25,6 +25,13 @@ uint32_t GetYear(const base::Time& time);
 
 uint64_t GetCurrentTimeStamp();
 
+base::TimeDelta GetRandomizedDelay(base::TimeDelta delay);
+
+base::TimeDelta GetRandomizedDelayWithBackoff(
```

Comment:
These helpers are used in `publisher_list_updater.cc` and will be used as we refactor timers to use the  timer classes from `base`.